session/session.go

```diff
@@ -194,7 +194,7 @@ type session struct {
 
 	store kv.Storage
 
-	parser *parser.Parser
+	parserPool *sync.Pool
```

Comment:
When there are too many `sync.Pool` objects, it would impose some side effect on Go GC.